uninstall.php

```diff
@@ -28,6 +28,24 @@
  * and to ensure only the site owner can perform this action.
  */
 if ( Constants::is_true( 'WC_REMOVE_ALL_DATA' ) ) {
+	/**
+	 * Load core packages autoloader.
+	 */
+	if ( version_compare( PHP_VERSION, '5.6.0', '>=' ) ) {
+		if ( ! class_exists( 'Automattic\WooCommerce\Admin\Install' ) ) {
+			require __DIR__ . '/src/Autoloader.php';
+
+			if ( ! \Automattic\WooCommerce\Autoloader::init() ) {
+				return;
+			}
+		}
+
+		// Hook in WooCommerce Admin installation code.
+		// @todo an alternate way to do this is allow FeaturePlugin::has_satisfied_dependencies()
+		// to work during uninstall - (no WooCommerce instance).
+		\Automattic\WooCommerce\Admin\Install::init();
```

Comment:
@woocommerce/isotope I think it suppose to be `drop_tables()` instead of `init()`.